parquet-column/src/main/java/org/apache/parquet/column/values/rle/RunLengthBitPackingHybridDecoder.java

```diff
@@ -56,6 +58,17 @@ public RunLengthBitPackingHybridDecoder(int bitWidth, ByteArrayInputStream in) {
     Preconditions.checkArgument(bitWidth >= 0 && bitWidth <= 32, "bitWidth must be >= 0 and <= 32");
     this.bitWidth = bitWidth;
     this.packer = Packer.LITTLE_ENDIAN.newBytePacker(bitWidth);
+    byte[] buf = new byte[in.available()];
+    in.read(buf, 0, in.available());
```

Comment:
it seems waste full to copy the content of the underlying buffer in a new byte[] when ByteArrayInputStream is backed by a byte[]. We may as well remove this constructor.